src/Framework/Numerical/Spline.h

```diff
@@ -145,7 +145,6 @@ class Spline : public TObject {
   TSpline5 * fInterpolator5;
   ROOT::Math::Interpolator * fGSLInterpolator;
   string     fInterpolatorType;
-  std::vector<double> fX, fY;
 
 ClassDef(Spline,1)
```

Comment:
You've changed the member data of the class; you'll want to bump the ClassDef version number.

Reply:
Fixed